scripts/BUILD.bazel

```diff
@@ -0,0 +1,42 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_binary", "kt_jvm_library", "kt_jvm_test")
+load("//scripts:script_assets.bzl", "generate_assets_list_from_text_protos")
+
+REGEX_PATTERN_CHECK_ASSETS = generate_assets_list_from_text_protos(
+    name = "script_assets",
+    file_content_validation_file_names = [
+        "file_content_validation_checks",
+    ],
+    filepath_pattern_validation_file_names = [
+        "filename_pattern_validation_checks",
+    ],
+)
+
+kt_jvm_binary(
+    name = "pattern_validation_check",
+    testonly = True,
+    data = REGEX_PATTERN_CHECK_ASSETS,
+    main_class = "org.oppia.android.scripts.RegexPatternValidationCheck$Companion",
+    runtime_deps = [":pattern_validation_lib"],
+)
+
+kt_jvm_library(
+    name = "pattern_validation_lib",
+    testonly = True,
+    srcs = glob(["src/java/org/oppia/android/scripts/*.kt"]),
```

Comment:
Can we avoid using glob? In general its better to explicitly list down the files rather than giving a generic rule like this

Reply:
Done.